lib/jekyll-last-modified-at/generator.rb

```diff
@@ -3,7 +3,7 @@ module LastModifiedAt
     class Generator < Jekyll::Generator
 
       def generate(site)
-        %w(posts pages docs_to_write).each do |type|
+        %w(posts pages static_files docs_to_write).each do |type|
           site.send(type).each do |item|
             item.data['last_modified_at'] = Determinator.new(site.source, item.path)
```

Comment:
In order to support future versions of Jekyll, keep this change and change this line to:

```ruby
item.data['last_modified_at'] = Determinator.new(site.source, item.path) if item.respond_to?(:data)
```